websites/R/Richup.io/presence.ts

```diff
@@ -0,0 +1,95 @@
+const presence = new Presence({
+		clientId: "1067910906912194590",
+	}),
+	pages: { [k: string]: string } = {
+		"/settings": "Settings",
+		"/store/player-appearance": "Player Appearance",
+		"/store/board-maps": "Board Maps",
+		"/store/upgrades": "Upgrades",
+		"/store/avatars": "Profile Pictures",
+		"/store/coins": "Coins",
+	};
+
+let startTimestamp: number;
+
+presence.on("UpdateData", async () => {
+	const presenceData: PresenceData = {
+			largeImageKey: "rio-logo",
+		},
+		{ pathname, href } = location,
+		settings = {
+			showJoinRoomButton: await presence.getSetting("join-room-button"),
+			showProfileButton: await presence.getSetting("profile-button"),
```

Comment:
no, do it like this.
https://github.com/PreMiD/Presences/blob/ce2965e9aea36106f4fe16398ecad2c3dcea9aa4/websites/T/Tesla/presence.ts#L7-L12

Reply:
alright